### PR TITLE
Adding footer code

### PR DESCRIPTION
**Pull request by evanhearne**
# Description

added footer code to website

## What

footer code has been placed into a separate file and all html files take code from there

## Why

To make it easier for future developers to edit footer code; it is stored in one central place.

## Verification Steps

What steps should be taken to verify that the change is working correctly

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by evanhearne** [comment]
@laurafitzgerald @denispop9 could you please review this when you have the time?

File: pages/about/policies.html [hunk 1]
```diff
@@ -180,7 +186,7 @@ <h1 class="ui header">Secondary School Literacy</h1>
             </div>
           </div>
         </div><!-- End Row 5 -->
-        <!-- Start Row 5 -->
+        <!-- Start Row 6 -->
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: @evanhearne same again here, this was added during the pr we worked through last night

File: pages/about/policies.html [hunk 2]
```diff
@@ -190,45 +196,28 @@ <h1 class="ui header">School Tour</h1>
               Read More <i class="right arrow icon"></i>
             </div>
           </div>
+          <div class="column">
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: same here

File: pages/about/policies.html [hunk 3]
```diff
@@ -437,6 +426,19 @@ <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
       </div>
     </div>
   </div>
+  <div class="ui modal auppolicy">
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: and here

File: pages/about/policies/aup.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
+<DOCTYPE! html>
```
Comment (laurafitzgerald) [inline pages/about/policies/aup.html]: This file also

**Comment by evanhearne** [comment]
@laurafitzgerald I will work on that today and ping you when it is fixed.

**Comment by laurafitzgerald** [comment]
@evanhearne it would be helpful for the team if you could pair with somebody as you work through this as a learning experience. As always ping if you need any assistance

**Comment by evanhearne** [comment]
@laurafitzgerald I remembered about how you can 'drop' commits, so that's what I did. I hope the changes have been sorted out, if you would like to do review it, please do so when you have the time. :)

**Comment by evanhearne** [comment]
Yes, @laurafitzgerald , I think that adding the navigation.html to the same directory sounds good.
I can understand what you said about the javascript and that there only needs to be one function for these two codes.
The reason why I did that for the footer code was so that I can just copy and paste. No other reason, I will adjust the footer code for consistency.

**Comment by evanhearne** [comment]
@laurafitzgerald changes have been made as you requested, please review when you have the time :)

**Comment by laurafitzgerald** [comment]
@evanhearne looks like the rebase went a little funny again on this one. Again let's be sure to cover this in the next session.

**Pull request merged**